decidim-admin/spec/commands/update_organization_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+require "spec_helper"
+
+module Decidim
+  module Admin
+    describe UpdateOrganization, :db do
+      describe "call" do
+        let(:organization) { create(:organization) }
+        let(:form) do
+          OrganizationForm.from_params(
+            organization: {
+              name: "My super organization",
+              description_en: "My description",
```

Comment:
Please don't couple to an actual set of languages. Use a test helper instead or just use english.